samples/miniupnpcplus/mapping/mapping.cpp

```diff
@@ -0,0 +1,42 @@
+/**
+ * \file mapping.cpp
+ * \author Sebastien Vincent <sebastien.vincent@freelan.org>
+ * \brief Test the UPnP functions.
+ */
+
+#include <iostream>
+
+#include <miniupnpcplus/upnp_device.hpp>
+
+int main(int argc, char** argv)
+{
+	static_cast<void>(argc);
+	static_cast<void>(argv);
+
+	try
+	{
+		miniupnpcplus::upnp_device dev(2000);
+		std::string tmp;
+
+		std::cout << "UPnP device created" << std::endl;
+
+		dev.register_port_mapping(miniupnpcplus::TCP, "12000", "12000",
```

Comment:
Currently all the reamining registered port are unregistered in upnp_device destructor.